parser/Cargo.toml

```diff
@@ -27,6 +27,8 @@ thiserror = "1.0.38"
 # binary.
 kanata-keyberon = { path = "../keyberon" }
 bytemuck = "1.15.0"
+colored = "2.1.0"
+num-format = "0.4.4"
```

Comment:
This does not seem worth adding as a dependency.

Reply:
so you'd be fine with errors like 
- `expected 0–18446744073709551615.` isntead of
- `expected 0–18,446,744,073,709,551,615`